info/v1/machine.go

```diff
@@ -14,6 +14,10 @@
 
 package v1
 
+import (
+	"github.com/google/cadvisor/utils/cloudinfo"
+)
```

Comment:
I'd suggest moving the datastructures reported in the API  (CloudProvider, InstanceType) here. That way, consumers of the api don't have to include utils/cloudinfo.